Source/Core/FeatureDetection.js

```diff
@@ -255,6 +253,15 @@ define([
         }
     }
 
+    var supportsCreateImageBitmapResult;
+    function supportsCreateImageBitmap() {
+        if (!defined(supportsCreateImageBitmapResult)) {
+            supportsCreateImageBitmapResult = defined(window.createImageBitmap);
```

Comment:
In general avoid using `window.` for globals because window isn't always the global object (for example, Node.js and workers).  You can just use `createImageBitmap`.  Since it's such a trivial check, there's no need to cache a value here, this whole function ca be simplified to:

```
    function supportsCreateImageBitmap() {
        return typeof createImageBitmap === 'function';
    }
```